app/views/posts/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>I am users post by id</h1>
\ No newline at end of file
+ <div class="container mt-5">
+  <div class="row justify-content-between m-auto border border-dark mb-1">
+    <h2><%= @post.title %> by <%= @post.author.name %></h2>
+    <p><%= @post.text %></p>
+    <span class="d-flex justify-content-end">Comments: <%= @post.commentscounter %>,Likes: <%= @post.likescounter %></span>
+  </div>
+  <% @post.comments.all.each do |comment| %>
+   <div class="row justify-content-between m-auto border border-dark mb-3">
+   <p><%= comment.author.name %>: <%= comment.text %></p>
+   </div>
+   <% end %>
+    </div>
+  
```

Comment:
| Your Post Show | Wireframe Post Show |
| --- | --- |
| ![image](https://github.com/sajibarafatsiddiqui/BlogMaster/assets/93039359/5df312a6-f678-47fc-a266-b06fb9ca7636) | ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/af51593a-4722-4c1b-a52f-0353d65a8daa) |

- [ ] It would be nice if you make the comments and likes counter to work to make your app work as expected :+1: right now you can see that there are two comments but the comments counter is not counting them :fearful:

Reply:
corrected